content/applications/inventory_and_mrp/maintenance/maintenance_calendar.rst

```diff
@@ -0,0 +1,378 @@
+Clicking this button opens a pop-up window titled :guilabel:`Select a spreadsheet to insert your
+pivot`.
+
+There are two tabs in this pop-up window: :guilabel:`Spreadsheets` and :guilabel:`Dashboards`. Click
+into one of these tabs and select a spreadsheet or dashboard in the database to add this pivot table
+to. Once ready, click :guilabel:`Confirm`. If this table shouldn't be added to a spreadsheet or
+dashboard, click :guilabel:`Cancel`.
+
+To the right of the :guilabel:`Insert in Spreadsheet` button are three buttons:
+
+- :guilabel:`Flip axis`: the x and y axis of the pivot data table flip.
+- :guilabel:`Expand all`: all the available rows and columns of the pivot data table expand fully.
+- :guilabel:`Download xlsx`: the pivot data table is downloaded as an .xlsx file.
+
+Graph view
+~~~~~~~~~~
+
+With the graph view selected, the following options appear between the search bar and visual
+representation of the data. These graph-specific options are located to the right of the
+:guilabel:`Measures` and :guilabel:`Insert in Spreadsheet` buttons.
+
+.. image:: maintenance_calendar/maintenance-calendar-graph-view-icons.png
+   :align: center
+   :alt: Graph type icons on Graph view page.
+
+There are three different types of graphs available to users to view the data:
+
+- :guilabel:`Bar Chart`: the data is displayed in a bar chart.
+- :guilabel:`Line Chart`: the data is displayed in a line chart.
+- :guilabel:`Pie Chart`: the data is displayed in a pie chart.
+
+When viewing the data as a :guilabel:`Bar Chart` graph, the data can be formatted in the following
+ways:
+
+- :guilabel:`Stacked`: the data is stacked on the graph.
+- :guilabel:`Descending`: the data is displayed in descending order.
+- :guilabel:`Ascending`: the data is displayed in ascending order.
+
+When viewing the data as a :guilabel:`Line Chart` graph, the data can be formatted in the following
+ways:
+
+- :guilabel:`Stacked`: the data is stacked on the graph.
+- :guilabel:`Cumulative`: the data is increasingly accumulated.
+- :guilabel:`Descending`: the data is displayed in descending order.
+- :guilabel:`Ascending`: the data is displayed in ascending order.
+
+When viewing the data as a :guilabel:`Pie Chart` graph, all relevant data is displayed by default,
+and no additional formatting options are available.
+
+Activity view
+~~~~~~~~~~~~~
+
+With the :guilabel:`Activity` view selected, all open maintenance requests are listed in their own
+row, with the ability to schedule activities related to those requests.
+
+.. image:: maintenance_calendar/maintenance-calendar-activity-view-type.png
+   :align: center
+   :alt: Maintenance requests on Activity view.
+
+Maintenance requests are listed in the :guilabel:`Maintenance Request` column as activities.
+Clicking a request opens a :guilabel:`Maintenance Request` popover that indicates the status of the
+request and the responsible *Technician*. To schedule an activity directly from the popover, click
```

Comment:
```suggestion
request, and the responsible technician. To schedule an activity directly from the pop-over window, click
```

Reply:
Partially implemented - everything except for popover which I am keeping as is per earlier related comment